src/oas.js

```diff
@@ -2,6 +2,7 @@
 const { pathToRegexp, match } = require('path-to-regexp');
 const getPathOperation = require('./lib/get-path-operation');
 const getUserVariable = require('./lib/get-user-variable');
+const getReference = require('./lib/get-reference');
```

Comment:
You should be able to use `findSchemaDefinition` in `utils.js` for this instead. Takes two args:

```
findSchemaDefinition(ref, oas);
```

Reply:
I saw that and it didn't look to map, but you are right, I can swap this out!